math/fibonacci.c

```diff
@@ -38,6 +39,42 @@ unsigned int fib(int number)
     return fib(number - 1) + fib(number - 2);
 }
 
+/**
+ * @brief Get the input from the user
+ * @return valid argument to the fibonacci function
+ */
+int getInput(void)
+{
+    int num, success = 0;
+    char buffer[4], *endPtr;
+
+    do
+    {
+        fgets(buffer, 4, stdin);  // Inputs the value from user
+
+        errno = 0;  // Reseting the errno value
+        num = strtol(buffer, &endPtr,
+                     10);  // Attempts to convert the string to integer
+
+        if (strlen(buffer) > 3 || num > 48)
+            success = 0;  // Three digit numbers are not allowed
+        else if (errno == ERANGE)
```

Comment:
This check is redundant, since only up to three characters are read from input, it's impossible for an overflow to happen, not to mention there's already a `num > 48` check